### ISSUE TITLE
Prophage search tool

### ISSUE THREAD

**Issue by curtisim0** [TAMU-CPT/CPT-ToolshedSource#12]
This was never migrated to the toolshed.

**Comment by jasonjgill** [comment]
Tested this again with a dummy sequence of 5 kb flanks of bacterial sequence with an intervening stretch of random DNA sequence, it worked to report this at the expected coordinates. Only tested with an XML file downloaded from NCBI BLAST. If the 25 column table feature doesn't work, can this be removed from the wrapper?

If this is ready to go I will do a PR to add it to usegalaxy.

**Comment by curtisim0** [comment]
Yes I can remove the 25 col feature. Feel free to feed me a input that you would expect to work and I can see if it...works or not.

How are you testing it?
